### PR TITLE
upgrade babel-eslint

### PR DESCRIPTION
**Pull request by djkirby**
There was an error with the previous version preventing
me from starting the app.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +59,7 @@
     "react": ">= 0.12.x"
   },
   "dependencies": {
+    "babel-eslint": "^5.0.0",
```
Comment (FrancescoCioria) [inline package.json]: Please move this to the `devDependencies` (we don't it to finish in the final bundle)

Comment (djkirby) [inline package.json]: ah yes :sweat:

**Comment by djkirby** [comment]
updated this @FrancescoCioria

**Comment by FrancescoCioria** [comment]
Great

**Pull request merged**